livecheck/main.py

```diff
@@ -462,15 +489,20 @@ def do_main(*, cat: str, ebuild_version: str, pkg: str, search_dir: Path,
             dn = Path(ebuild).parent
             log.debug('%s -> %s', ebuild, new_filename)
             if settings.keep_old.get(cp, not settings.keep_old_flag):
-                if settings.git_flag:
-                    try:
+                try:
+                    if settings.git_flag:
                         sp.run(('git', 'mv', ebuild, new_filename), check=True)
-                    except sp.CalledProcessError:
-                        log.exception('Error moving %s to %s.', ebuild, new_filename)
-                        return
-                else:
-                    sp.run(('mv', ebuild, new_filename), check=True)
-            Path(new_filename).write_text(content, encoding='utf-8')
+                    else:
+                        sp.run(('mv', ebuild, new_filename), check=True)
+                except sp.CalledProcessError:
+                    log.exception('Error moving %s to %s.', ebuild, new_filename)
+                    return
```

Comment:
Let's just use [`shutil.copyfile`](https://docs.python.org/3/library/shutil.html#shutil.copyfile).

```suggestion
                        copyfile(ebuild, new_filename)
                except OSError:
                    log.exception('Error moving `%s` to `%s`.', ebuild, new_filename)
                    return
```